src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -2688,39 +2878,28 @@ void AutomationInstrumentClipView::initParameterSelection() {
 
 	initPadSelection();
 
-	clip->lastSelectedParamID = kNoLastSelectedParamID;
+	clip->lastSelectedParamID = kNoSelection;
 	clip->lastSelectedParamKind = Param::Kind::NONE;
-	clip->lastSelectedParamShortcutX = kNoLastSelectedParamShortcut;
-	clip->lastSelectedParamShortcutY = kNoLastSelectedParamShortcut;
+	clip->lastSelectedParamShortcutX = kNoSelection;
+	clip->lastSelectedParamShortcutY = kNoSelection;
 	clip->lastSelectedParamArrayPosition = 0;
 
+	//if we're going back to the Automation Overview, set the display to show "Automation Overview"
+	//and update the knob indicator levels to match the master FX button selected
 	display->cancelPopup();
-
-	//if we're going back to the Automation Overview, set the display to show Midi Channel again (7seg only)
-	if (instrument->type == InstrumentType::MIDI_OUT) {
-
-		if (display->have7SEG()) {
-			if (((MIDIInstrument*)instrument)->channel < 16) {
-				display->setTextAsSlot(((MIDIInstrument*)instrument)->channel + 1,
-				                       ((MIDIInstrument*)instrument)->channelSuffix, false, false);
-			}
-			else {
-				char const* text =
-				    (((MIDIInstrument*)instrument)->channel == MIDI_CHANNEL_MPE_LOWER_ZONE) ? "Lower" : "Upper";
-				display->setText(text, false, 255, false);
-			}
-		}
-	}
+	renderDisplay();
+	view.setKnobIndicatorLevels();
```

Comment:
need to restore knob indicator LED's to display current values for parameters assigned to mod encoders on automation overview